tox.ini

```diff
@@ -6,15 +6,17 @@ deps =
     flake8
     isort
     pytest
+    future
 commands =
     py.test {posargs}
-    flake8
+    flake8 qstring tests
```

Comment:
I'd prefer excluding the problematic files/directories instead of whitelisting. This would prevent flake8 from linting `setup.py` and `docs/conf.py` for example. Also, I'd like to have this sort of configuration in `[flake8]` section in `tox.ini` so that it's applied regardless of the way you invoke flake8.